src/process/releases.md

```diff
@@ -3,3 +3,56 @@ title: Releases
 type: process
 order: 510
 ---
+
+We aim to follow the Gitflow strategy of deploying sites in combination with CI/CD.
+
+## Steps
+
+To deploy a site of system, you can roughly follow these steps:
+
+1. In Jira, Identify tickets ready to release
+    a. Issues
+    b. Search for Issues
+    c. Set project to the one you want to deploy
+    d. Set status to Ready to Release
+2. Create a release version
+    a. Projects > Project to be released
+    b. Releases (in side bar)
+    c. Create version, based on semantic versioning with details of that release
+3. Tag tickets ready to release, easiest way to so this
+    a. Go back to current search in step 1
+    b. Tools > Bulk change (top right)
+    c. Select all issues to be released > Next
+    d. Edit issues > Next
+    e. Change fix version > "replace all with" > set version > next
+    f. Confirm
+4. In VSTS, create branch from `develop` called `release/{your version number (no v. prefix)}`
+    - CD should step in and the application should be built & deployed to dev then staging.
```

Comment:
maybe a little comment to say that it depends on automatic build/release triggers being set up for that project?